content/intro/overview.md

```diff
@@ -0,0 +1,34 @@
+---
+linkTitle: अवलोकन
+title: Brigade की अवलोकन
+description: Brigade का उच्च स्तरीय दृश्य
+section: परिचय
+weight: 1
+aliases: 
+  - /overview.md
+  - /intro/overview.md
+---
+
+Brigade एक इवेंट संचालित स्क्रिप्टिंग टूल है जो Kubernetes पर चलता है। इसका मतलब यह है:
+
+- Brigade क्लाउड में स्क्रिप्ट योग्य स्वचालित कार्यों को चलाने के लिए है।
+- Brigade के लिए आपको कोई होस्ट सर्वर प्रबंधित करना ज़रुरी नहीं है।
+-Brigade विशेष रूप से सीआई और सीडी वर्कलोड के लिए उपयुक्त है जैसे:
+- Automated testing
+  - GitHub hook integration
+  - Building artifacts and releases
+  - Managing deployments
+- Brigade अपने API server के साथ रहता है |
+  - बातचीत सीधे ब्रिगेड के माध्यम से होती है, `brig` CLI या SDK के माध्यम से
+  - ब्रिगेड स्थापना का प्रबंधन करने वाले ऑपरेटर अभी भी ब्रिगेड संसाधनों की जांच के लिए K8s API पर जा सकते हैं।
+  - ब्रिगेड सर्वर को हेल्म के साथ तैनात और प्रबंधित किया जाता है, इस प्रकार होस्टिंग किसी भी स्टॉक कुबेरनेट्स सिस्टम पर हो सकती है, एज़्योर से मिनिक्यूब तक।
+- Brigade authn/authz की चिंता को संभलता है |
+  - तृतीय-पक्ष प्रमाणीकरण विकल्पों में GitHub का OAuth2 पहचान प्रदाता और OIDC- अनुरूप समाधान जैसे Azure Active Directory और Google Identity Platform शामिल हैं।
+  - Brigade की प्रशासकों को प्राधिकरण,Service Account, Role और User management पे पूरा नियंत्रण है |
+- Brigade कुछ जान पहचान के टेक्नोलॉजीज को इस्तमाल करके इवेंट हैंडलिंग को आसान बनाना देता है:
+  - प्रोजेक्ट डेवलपर मूल JavaScript या TypeScript में स्क्रिप्ट लिखते हैं
+  (Node.js सीखने की ज़रुरत नहीं)
+  - ईवेंट वर्कलोड को संभालने के लिए OCI/Docker इमेज का उपयोग करें
+  - प्रोजेक्ट कॉन्फ़िगरेशन YAML में लिखा गया है और आपके पसंदीदा VCS के माध्यम से आसानी से बना रहता है
+
+Brigade को टीमों के लिए डिज़ाइन किया गया है। यह SaaS नहीं है, न ही इसे कॉन्फ़िगर करने और चलाने के लिए डोमेन विशेषज्ञों की एक बड़ी संख्या की आवश्यकता है। इसके बजाय, इसे स्थापित करना, कॉन्फ़िगर करना और बनाए रखना आसान होना चाहिए।
```

Comment:
"यह कोई SaaS नहीं है".

"न ही इसे कॉन्फ़िगर करने और चलाने के लिए डोमेन विशेषज्ञों की एक सेना (or you can use समूह) की आवश्यकता है।" -> let's instead use

Reply:
Okay

Reply:
So, will it not be like ignoring the the fact that it is not a SaaS?

Reply:
Sorry, I didn't get your question?

Reply:
Are you saying that I should totally delete the sentence यह कोई SaaS नहीं है ?

Reply:
oh sorry my bad, thank you:)

Reply:
No no, I meant "यह SaaS नहीं है" -> "यह **कोई** SaaS नहीं है"

Reply:
Yeah, I got it. Thank you.